src/regdesk/printer_manager.js

```diff
@@ -0,0 +1,66 @@
+import { LabelEpl, LP2844 } from 'WebZLP/src/LP2844'
+
+export class PrinterManager {
+    #devices = [];
+    #printers = [];
```

Comment:
Could make this a map? Or at least a list of objects? Would ensure you never have the two get out of order relative to each other.

Reply:
Took a slightly different approach, let me know what you think.